jarviscli/plugins/cricket.py

```diff
@@ -115,6 +115,9 @@ def scorecard(self, desc):
     def score(self, jarvis):
         matches = self.all_matches()
         jarvis.say(Fore.RED + "\nALL MATCHES\n" + Fore.LIGHTBLUE_EX)
+        if matches == []:
+            jarvis.say(Fore.BLACK + "No Matches Being Played!\n" + Fore.RESET)
```

Comment:
Black is a quite sub-optimal color - since black or really dark colors are quite common for terminal backgrounds.

**Thanks for fixing this bug!**



By the way; jarvis.say() also has an option for color:
```suggestion
            jarvis.say("No Matches Being Played!\n", Fore.RED)
```
But this option isn't used in this whole module anyway ;).